tools/ktx/command_create.cpp

```diff
@@ -1329,6 +1350,16 @@ void CommandCreate::executeCreate() {
                 image->yflip();
             }
 
+            if (options.normalize) {
+                if (target.format().transfer() != KHR_DF_TRANSFER_LINEAR) {
+                    fatal(rc::INVALID_FILE,
+                        "Input file \"{}\" transfer function is not linear. Normalize is only available for linear images. "
+                        "Use --assign-oetf=linear or --convert-oetf=linear to convert to linear if required.",
```

Comment:
```
"Use or modify {} or {} settings to assign the transfer function or convert the input image to linear, if required.",
OptionsCreate::kAssignOetf, OptionsCreate::kConvertOetf);
```

I'm not entirely happy with this wording.

Regarding this and the comment above, if you want to get really ambitious you can check `firstInputFile.format().transfer()` to see it is the file or the options and modify the error message accordingly.

Reply:
I have rewored this area with more detailed error message https://github.com/KhronosGroup/KTX-Software/pull/977/commits/ae46489470217b430e1f957aa3c4a5401d90fa69